### ISSUE TITLE
Question about the depth distortion loss

### ISSUE THREAD

**Issue by ZhuoxiaoLi** [hbb1/2d-gaussian-splatting#37]
Hi, thanks for your excellent work!

I noticed that you set the lambda_dist = 0.0 for the distortion loss in the latest version, and the last vision was set to 100.

May I ask why you set the "dist_loss = lambda_dist * (rend_dist).mean()" = 0 during the entire training iteration, and will it cause the final rendered depth to look bad?

**Comment by hbb1** [comment]
The depth dist regularization should be tuned according to the scene's scale, otherwise the results can be worse, see #21 . 
Therefore, I set the default weight to be zero and leave it for users for tuning a rendered depth map when necessary.
